shell/pages/c/_cluster/apps/charts/chart.vue

```diff
@@ -677,10 +681,17 @@ export default {
         &--keywords{
           word-break: break-word;
           color: var(--link-text-secondary);
+          padding-bottom: 0;
 
           .keyword-links {
-            a {
-              margin-right: 8px;
+            .keyword-item {
+              display: inline-block;
+              white-space: nowrap;
+            }
+
+            span {
+              margin-right: 4px;
+              margin-left: -2px;
             }
```

Comment:
I think we want to move this styling into the .keyword-item selector. With it being here it gets applied to the both the keyword-item and the span that contains the comma.